validator/src/main/resources/expected-data-template/eks/aws-sdk-call-metric.mustache

```diff
@@ -136,7 +136,38 @@
       value: AWS.SDK.S3
     -
       name: RemoteTarget
-      value: ::s3:::e2e-test-bucket-name
+      value: {{remoteTargetName}}
+
+-
+  metricName: Latency
+  namespace: {{metricNamespace}}
+  dimensions:
+    -
+      name: Service
+      value: {{serviceName}}
+    -
+      name: HostedIn.EKS.Cluster
+      value: {{platformInfo}}
+    -
+      name: HostedIn.K8s.Namespace
+      value: {{appNamespace}}
+    -
+      name: RemoteService
+      value: AWS.SDK.S3
+    -
+      name: RemoteTarget
+      value: {{remoteTargetName}}
+
+-
+  metricName: Latency
+  namespace: {{metricNamespace}}
+  dimensions:
+    -
+      name: RemoteService
+      value: AWS.SDK.S3
+    -
+      name: RemoteTarget
+      value: {{remoteTargetName}}
```

Comment:
Seems like we are not checking for just `[RemoteTarget]`. Is that expected?

Reply:
Yes, today we are not rolling up metrics into just `[RemoteTarget]`.
https://github.com/jj22ee/amazon-cloudwatch-agent/blob/main/translator/translate/otel/exporter/awsemf/appsignals_config_eks.yaml#L29